modules/ROOT/pages/_partials/include-alerts-config.adoc

```diff
@@ -53,16 +53,16 @@ TODO _FUTURE?
 [[notifications]]
 === Notifications
 
-Configure a notification to email when an alert is triggered.
+Configure a notification to email when an advanced alert is triggered.
 
 [%header,cols="1,4"]
 |===
-2+| Notification Settings for Alerts
-| `Send To` | Email address for a recipient of the alert notification. You can add more than one email recipient to the list.
+2+| Notification Settings for Advanced Alerts
+| `Send To` | Email address for a recipient of the advanced alert notification. You can add more than one email recipient to the list.
```

Comment:
Ah, I see. If even one entry in a group is a complete sentence, then all take periods.
But it is best to try to make them all sentences or all fragments, if possible....